pkg/apis/core/types.go

```diff
@@ -3480,12 +3480,23 @@ const (
 	ServiceExternalTrafficPolicyTypeCluster ServiceExternalTrafficPolicyType = "Cluster"
 )
 
+// These are valid conditions of a service.
+const (
+	// LoadBalancerPortsNotConfiguredSuccessfully represents the status of the requested ports
+	// on the same cloud Load Balancer instance.
+	LoadBalancerPortsNotConfiguredSuccessfully = "LoadBalancerPortsNotConfiguredSuccessfully"
```

Comment:
sorry to pick: "LoadBalancerPortsNotConfiguredSuccessfully" vs "LoadBalancerPortsError"  ?

Reply:
Yep, no problem. It supposed to be a nice API :) Fixed.